agent/config/file_watcher_windows.go

```diff
@@ -0,0 +1,25 @@
+//go:build windows
+// +build windows
+
+package config
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+)
+
+func (w *FileWatcher) getFileId(filename string) (uint64, error) {
```

Comment:
Windows don't have an Inode so we have a special implementation that use `CreationTime`